manifests/kubeedge/output-prometheusRemoteWrite.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterOutput
+metadata:
+  name: prometheus-remote-write
+  labels:
+    fluentbit.fluent.io/enabled: "true"
+    node-role.kubernetes.io/edge: "true"
+spec:
+  matchRegex: (?:kubeedge|service)\.(.*)
+  prometheusRemoteWrite:
+    host: <cloud-prometheus-service-host>
+    port: <cloud-prometheus-service-port>
+    uri: /api/v1/write
+    addLabels : 
```

Comment:
The labels you added are for fluent bit plugin example, should be changed to a more meaningful one
```yaml
    addLabels : 
      app: fluentbit
      job: kubeedge
```